app/assets/stylesheets/dashboards.scss

```diff
@@ -33,3 +33,8 @@
   color: red;
   padding: 0px 10px 0px 10px;
 }
+
+.remove-all-patients-link {
```

Comment:
Can we do this with bootstrap styling rather than a one-off CSS class? Assigning the classes `text-danger` and `pull-right` should do the trick.